Some of the test fail when there is no network

Some of the tests in `reproject/interpolation` use remote data and fail if there is no network connection:

```sh
pytest reproject/interpolation
```

```py
================================== test session starts ===================================
platform darwin -- Python 3.9.9, pytest-7.2.1, pluggy-1.0.0

Running tests with reproject version 0.10.dev87+g185dae10.
Running tests in reproject/interpolation.

Date: 2023-02-24T10:57:01

Platform: macOS-10.16-x86_64-i386-64bit

Executable: /Users/smaret/Documents/Software/reproject/.direnv/python-3.9.9/bin/python3

Full Python Version: 
3.9.9 (v3.9.9:ccb0e6a345, Nov 15 2021, 13:29:20) 
[Clang 6.0 (clang-600.0.57)]

encodings: sys: utf-8, locale: UTF-8, filesystem: utf-8
byteorder: little
float info: dig: 15, mant_dig: 15

Package versions: 
Numpy: 1.24.2
Scipy: 1.10.1
Astropy: 5.2.1
astropy-healpix: 0.7
Cython: not available

Using Astropy options: remote_data: none.

rootdir: /Users/smaret/Documents/Software/reproject, configfile: setup.cfg
plugins: remotedata-0.4.0, hypothesis-6.68.2, mock-3.10.0, filter-subpackage-0.1.2, astropy-header-0.2.2, doctestplus-0.12.1, astropy-0.10.0, cov-4.0.0, openfiles-0.5.0, arraydiff-0.5.0
collected 107 items                                                                      

reproject/interpolation/tests/test_core.py ....................................... [ 36%]
...............ss......................................FFFFFFFFFFFF.               [100%]

======================================== FAILURES ========================================
_____________________ test_blocked_against_single[block_size0-True] ______________________
```

--- a/reproject/interpolation/tests/test_core.py
+++ b/reproject/interpolation/tests/test_core.py
@@ -25,6 +25,7 @@ def as_high_level_wcs(wcs):
 @pytest.mark.array_compare(single_reference=True)
 @pytest.mark.parametrize("wcsapi", (False, True))
 @pytest.mark.parametrize("roundtrip_coords", (False, True))
+@pytest.mark.remote_data
 def test_reproject_celestial_2d_gal2equ(wcsapi, roundtrip_coords):
     """
     Test reprojection of a 2D celestial image, which includes a coordinate
@@ -65,6 +66,7 @@ for wcsapi in (False, True):
 @pytest.mark.array_compare(single_reference=True)
 @pytest.mark.parametrize(("wcsapi", "axis_order"), tuple(COMBINATIONS))
 @pytest.mark.parametrize("roundtrip_coords", (False, True))
+@pytest.mark.remote_data
 def test_reproject_celestial_3d_equ2gal(wcsapi, axis_order, roundtrip_coords):
     """
     Test reprojection of a 3D cube with celestial components, which includes a
@@ -118,6 +120,7 @@ def test_reproject_celestial_3d_equ2gal(wcsapi, axis_order, roundtrip_coords):
 @pytest.mark.array_compare(single_reference=True)
 @pytest.mark.parametrize("wcsapi", (False, True))
 @pytest.mark.parametrize("roundtrip_coords", (False, True))
+@pytest.mark.remote_data
 def test_small_cutout(wcsapi, roundtrip_coords):
     """
     Test reprojection of a cutout from a larger image (makes sure that the
@@ -151,6 +154,7 @@ def test_small_cutout(wcsapi, roundtrip_coords):
 
 
 @pytest.mark.parametrize("roundtrip_coords", (False, True))
+@pytest.mark.remote_data
 def test_mwpan_car_to_mol(roundtrip_coords):
     """
     Test reprojection of the Mellinger Milky Way Panorama from CAR to MOL,
@@ -183,6 +187,7 @@ def test_mwpan_car_to_mol(roundtrip_coords):
 
 
 @pytest.mark.parametrize("roundtrip_coords", (False, True))
+@pytest.mark.remote_data
 def test_small_cutout_outside(roundtrip_coords):
     """
     Test reprojection of a cutout from a larger image - in this case the
@@ -208,6 +213,7 @@ def test_small_cutout_outside(roundtrip_coords):
 
 
 @pytest.mark.parametrize("roundtrip_coords", (False, True))
+@pytest.mark.remote_data
 def test_celestial_mismatch_2d(roundtrip_coords):
     """
     Make sure an error is raised if the input image has celestial WCS
@@ -234,6 +240,7 @@ def test_celestial_mismatch_2d(roundtrip_coords):
 
 
 @pytest.mark.parametrize("roundtrip_coords", (False, True))
+@pytest.mark.remote_data
 def test_celestial_mismatch_3d(roundtrip_coords):
     """
     Make sure an error is raised if the input image has celestial WCS
@@ -270,6 +277,7 @@ def test_celestial_mismatch_3d(roundtrip_coords):
 
 
 @pytest.mark.parametrize("roundtrip_coords", (False, True))
+@pytest.mark.remote_data
 def test_spectral_mismatch_3d(roundtrip_coords):
     """
     Make sure an error is raised if there are mismatches between the presence
@@ -334,6 +342,7 @@ def test_naxis_mismatch(roundtrip_coords):
 
 
 @pytest.mark.parametrize("roundtrip_coords", (False, True))
+@pytest.mark.remote_data
 def test_slice_reprojection(roundtrip_coords):
     """
     Test case where only the slices change and the celestial projection doesn't
@@ -383,6 +392,7 @@ def test_slice_reprojection(roundtrip_coords):
 
 
 @pytest.mark.parametrize("roundtrip_coords", (False, True))
+@pytest.mark.remote_data
 def test_inequal_wcs_dims(roundtrip_coords):
     inp_cube = np.arange(3, dtype="float").repeat(4 * 5).reshape(3, 4, 5)
     header_in = fits.Header.fromtextfile(
@@ -406,6 +416,7 @@ def test_inequal_wcs_dims(roundtrip_coords):
 
 
 @pytest.mark.parametrize("roundtrip_coords", (False, True))
+@pytest.mark.remote_data
 def test_different_wcs_types(roundtrip_coords):
 
     inp_cube = np.arange(3, dtype="float").repeat(4 * 5).reshape(3, 4, 5)
@@ -435,6 +446,7 @@ def test_different_wcs_types(roundtrip_coords):
 
 
 @pytest.mark.parametrize("roundtrip_coords", (False, True))
+@pytest.mark.remote_data
 def test_reproject_3d_celestial_correctness_ra2gal(roundtrip_coords):
 
     inp_cube = np.arange(3, dtype="float").repeat(7 * 8).reshape(3, 7, 8)
@@ -476,6 +488,7 @@ def test_reproject_3d_celestial_correctness_ra2gal(roundtrip_coords):
 
 
 @pytest.mark.parametrize("roundtrip_coords", (False, True))
+@pytest.mark.remote_data
 def test_reproject_with_output_array(roundtrip_coords):
     """
     Test both full_reproject and slicewise reprojection. We use a case where the
@@ -510,6 +523,7 @@ def test_reproject_with_output_array(roundtrip_coords):
 
 @pytest.mark.array_compare(single_reference=True)
 @pytest.mark.parametrize("file_format", ["fits", "asdf"])
+@pytest.mark.remote_data
 def test_reproject_roundtrip(file_format):
 
     # Test the reprojection with solar data, which ensures that the masking of
@@ -578,6 +592,7 @@ def test_reproject_roundtrip(file_format):
 
 
 @pytest.mark.parametrize("roundtrip_coords", (False, True))
+@pytest.mark.remote_data
 def test_identity_with_offset(roundtrip_coords):
 
     # Reproject an array and WCS to itself but with a margin, which should
@@ -708,6 +723,7 @@ def test_blocked_broadcast_reprojection(input_extra_dims, output_shape, parallel
 
 @pytest.mark.parametrize("parallel", [True, 2, False])
 @pytest.mark.parametrize("block_size", [[40, 40], [500, 500], [500, 100], None])
+@pytest.mark.remote_data
 def test_blocked_against_single(parallel, block_size):
 
     # Ensure when we break a reprojection down into multiple discrete blocks
@@ -753,6 +769,7 @@ def test_blocked_against_single(parallel, block_size):
     np.testing.assert_allclose(footprint_test, footprint_reference, equal_nan=True)
 
 
+@pytest.mark.remote_data
 def test_blocked_corner_cases():
 
     """

